### ISSUE TITLE
Parse zip for v2 compatible src json

### ISSUE THREAD

**Issue by tjklemz** [neutrinog/d43-catalog#2]
1. Download zip (which is the zipped repository)
2. Parse chunks
3. Grab manifest info (issued == published), such as checking level
4. Create src json(s) and sign it (?)

For Bible, for each book, dump USFM from Zip into a bucket and get sigs.
For OBS, parse the data into a src json. Also read app_words json from Zip and combine into the src json. E.g.
- https://api.unfoldingword.org/obs/txt/1/en/obs-en.json
- https://git.door43.org/Door43-Catalog/en_obs/src/master/.apps/ufw

**Comment by neutrinog** [comment]
@tjklemz I've fixed the v3 api so it includes the checking info within the resource.

```json
"checking": {
  "checking_entity": [
     "translation team"
  ],
  "checking_level": "1"
}
```

**Comment by neutrinog** [comment]
I also added `comment` to the resources.

**Comment by neutrinog** [comment]
This issue is closed in favor of #4
